### ISSUE TITLE
Parallel execution

### ISSUE THREAD

**Issue by AlexDaniel** [perlpilot/p6-sake#9]
For example, rake supports parallel execution of tasks. I can't see why sake written in Perl 6 wouldn't, given how easy it is to parallelize stuff in Perl 6.

**Comment by wbiker** [comment]
Basically, a good idea. However, I use sake and parallelize stuff already in the tasks. And I use task dependencies which are expected to run one after another.

**Comment by AlexDaniel** [comment]
@wbiker yeah, I think what I have in mind is compatible with that. I'm actually working on a slight rewrite right now but I keep getting sidetracked… Anyway, in what I have, you can return a Promise from a task (so a `start` block is fine), and sake will move on doing other tasks that don't depend on it. I think that will do what you want.

**Comment by wbiker** [comment]
I see. That would be cool!
